devdocs/settingUp.md

```diff
@@ -53,6 +55,9 @@ Important: When a version is specified, please install that version instead of t
    * `src/main/webapp/WEB-INF/appengine-web.xml`<br>
    Create it using `appengine-web.template.xml`. 
    For now, property values can remain as they are.
+   * `.settings/com.google.gdt.eclipse.core.prefs`<br>
+   Create it using `com.google.gdt.eclipse.core.template.prefs`.
+   Change all `*` in `jarsExcludedFromWebInfLib` to your TEAMMATES project folder, e.g. `C:/TEAMMATES`.
```

Comment:
For a newbie, it may not be clear what this means. May be give the modified line as an example?